Tests/SonarScanner.MSBuild.PreProcessor.Test/EmbeddedAnalyzerInstallerTests.cs

```diff
@@ -172,27 +160,21 @@ public void EmbeddedInstall_MultiplePlugins_Succeeds()
         }
 
         [TestMethod]
-        public void EmbeddedInstall_MissingResource_SucceedsWithWarningAndNoFiles()
+        public void EmbeddedInstall_MissingResource_ThrowFileNotFoundException()
         {
+            const string missingPluginKey = "missingPlugin";
+            const string missingPluginVersion = "1.0";
+            const string missingPluginResource = "resource.txt";
```

Comment:
Personal opinion: I liked `could.be.anything`. It conveys the information that the name of the plugin resource is not relevant for the test... and could be anything!